test-common/src/test/java/org/triplea/test/common/IpAddressMatchersTest.java

```diff
@@ -0,0 +1,80 @@
+package org.triplea.test.common;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.triplea.test.common.IpAddressMatchers.ipAddressMatching;
+
+import java.net.InetAddress;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class IpAddressMatchersTest {
+
+  private static final String IP = "99.99.99.99";
+
+  @Test
+  @DisplayName("Verify null IP as value will not match")
+  void nullValueNeverMatches() throws Exception {
+    assertThrows(AssertionError.class, () -> assertThat(null, is(ipAddressMatching(IP))));
+    assertThrows(
+        AssertionError.class,
+        () -> assertThat(null, is(ipAddressMatching(InetAddress.getByName(IP)))));
```

Comment:
I get that you're testing the matcher here, but would just negating the matcher do the same job? (I'm pretty sure that works for custom matchers as well.)

Reply:
The matcher was useful for some time, eventually it was no longer needed. I did not realize it was still included. I removed it: 7e4e750e9

The negation probably would work. I think I'd probably want to see both tests to make sure there are no surprises. After-all, part of the behavior is that we *will* throw a certain exception if the matcher fails. The negation should work as we're just adding an implementation of a `Matcher`, it's the same reason that `is` works as well.

All bit of a moot point though as this was unused in the end.